### PR TITLE
Add missing constructor properties to typing.

### PR DESCRIPTION
**Pull request by MugeSo**

### CODE REVIEW

File: decimal.d.ts [hunk 1]
```diff
@@ -523,6 +523,38 @@ export declare class Decimal {
...
+    // E.g. `Decimal.config({ precision: 20 });`
+    static readonly precision: number;
+
+    // The rounding mode used by default by `toInteger`, `toDecimalPlaces`, `toExponential`,
+    // `toFixed`, `toPrecision` and `toSignificantDigits`.
+    //
+    // E.g.
+    // `Decimal.rounding = 4;`
+    // `Decimal.rounding = Decimal.ROUND_HALF_UP;`
+    static rounding: number;
```
Comment (MikeMcl) [inline decimal.d.ts]: Should this also have been `readonly`?

Comment (MugeSo) [reply decimal.d.ts]: It should be not `readonly` to allow the e.g. code `Decimal.rounding = 4;`

Comment (MikeMcl) [reply decimal.d.ts]: Yes, but the same could be said for `precision`, `toExpNeg`, `toExpPos` and `LN10`.

I think it is best to have them all __not__ `readonly`, with `readonly` only for the rounding modes, i.e. `ROUND_UP` etc.

Comment (MugeSo) [reply decimal.d.ts]: fixed in 5cd4531

Comment (MikeMcl) [reply decimal.d.ts]: Okay, thanks.

**Pull request merged**